Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_-_Get_Endpoint_Forensics_Data.yml

```diff
@@ -197,14 +197,14 @@ tasks:
     task:
       id: fc7c593b-2e96-40de-8d2e-1a29f22325d3
       version: -1
-      name: Check a device was provided
-      description: Check a device was provided
+      name: Check if device id was provided
```

Comment:
ID, not id.

also - you can remove the was. Check if device ID provided.

Reply:
Done. Note that I have required the same request form in Cortex XDR as well.